src/frontend/efiling-demo/README.md

```diff
@@ -4,9 +4,15 @@ Welcome to the eFiling demo!
 
 ## Running the Frontend
 
+In the project directory (`efiling-demo`):
+
 Set the required environment variables locally. Create a `.env.development` file and populate the environment variables as shown in the `.env.example` file.
 
-In the project directory (`efiling-demo`), you can run:
+In the root directory (`jag-file-submission`), run:
+
+### `docker-compose up efiling-api efiling-frontend keycloak keycloak-config redis`
```

Comment:
```suggestion
```bash
docker-compose up efiling-api efiling-frontend keycloak keycloak-config redis
```
```

cmd should be wrapped in ```